example/harris.cpp

```diff
@@ -1,12 +1,13 @@
 //
 // Copyright 2019 Olzhas Zhumabek <anonymous.from.applecity@gmail.com>
+// Copyright 2021 Pranam Lashkari <plashkari628@gmail.com>
 //
 // Use, modification and distribution are subject to the Boost Software License,
 // Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
 // http://www.boost.org/LICENSE_1_0.txt)
 //
 #include <boost/gil/extension/io/png.hpp>
-#include <boost/gil/extension/numeric/convolve.hpp>
+#include <boost/gil/image_processing/convolve.hpp>
```

Comment:
I guess this was "find & replace" change, but it would be good to sort the includes though.

e.g. here move `convolve.hpp` below `image.hpp`

Reply:
Thanks for the reminder. in this case, I can reformat the entire files to clang-format when moving around :D

Reply:
Sure